x-pack/platform/plugins/shared/alerting/server/plugin.ts

```diff
@@ -343,6 +345,42 @@ export class AlertingPlugin {
     });
     this.ruleTypeRegistry = ruleTypeRegistry;
 
+    plugins.taskManager.registerTaskDefinitions({
+      taskWithApiKey: {
+        title: 'taskWithApiKey',
+        createTaskRunner: ({ taskInstance }) => ({
+          run: async () => {
+            const services = await core.getStartServices();
+
+            const fakeRawRequest: FakeRawRequest = {
+              headers: {
+                authorization: `ApiKey ${taskInstance.userScope?.apiKey}`,
+              },
+              path: '/',
+            };
+
+            const path = addSpaceIdToPath('/', taskInstance.userScope?.spaceId || 'default');
+
+            // Fake request from the API key
+            const fakeRequest = kibanaRequestFactory(fakeRawRequest);
+            services[0].http.basePath.set(fakeRequest, path);
+
+            // Getting access to scoped clients using the API key
+            // const scopedClusterClient = services[0].elasticsearch.client.asScoped(fakeRequest);
+            // const savedObjectsClient = services[0].savedObjects.getScopedClient(fakeRequest);
+
+            console.log(`TASK WITH API KEY - ${taskInstance.id}`);
```

Comment:
I think these can just be converted to use a logger ... but maybe we want something more "visual", somehow?